docs/deployment/sr_operator.md

```diff
@@ -151,12 +151,12 @@ From outside the Kubernetes cluster, you can access the StarRocks cluster throug
 
     ```YAML
     starRocksFeSpec:
-        image: starrocks/alpine-fe:2.4.1
-        replicas: 3
-        requests:
+      image: starrocks/fe-ubuntu:latest
```

Comment:
if here uses `latest`, and then upgrade to `3.0-latest`, it is actually a downgrade giving current release.